assets/static/js/dataset-submission.js

```diff
@@ -257,7 +280,9 @@ $(function() {
                 }
             },
             error: function(jqXHR, textStatus, errorThrown) {
-                if ([400, 401].includes(jqXHR.status)) {
```

Comment:
extra logic?

Reply:
It's not really needed now, but I left it, incase we want to add to this list or codes. But it could be changed to `401 === jqXHR.status` add a code suggestion if you feel it should be changed.